README.md

```diff
@@ -267,6 +267,46 @@ dependencies:
   - someDep
 ```
 
+## Editor Integration
+
+Note to editor integrators: there is now a haskell-language-server-wrapper executable, which is installed alongside the haskell-language-server executable. When this is invoked in the project root directory, it attempts to work out the GHC version used in the project, and then launch the matching hie executable.
+
+All of the editor integrations assume that you have already installed HLS (see above) and that stack put the hls binary in your path (usually ~/.local/bin on linux and macOS).
```

Comment:
Binary is named `haskell-language-server`, maybe it is confusing to newcomers if they read `hls binary`?